File: Java/Examples/ErrorFilterTest/ErrorFilterTest/MainClass.java
package ErrorFilterTest;

public class MainClass
{
	public static void main(String[] args)
	{
		Subject s = new Subject();
				
		s.f1();	
		try
		{						
			s.f2(); // should fail at runtime
		}
		catch (Exception e)
		{
			System.out.println("Exception caught with message: " + e.getMessage());
		}
	}
}

File: StarLight/Examples/TestCases/ErrorFilterTest/MainClass.cs
using System;

namespace ErrorFilterTest
{
	class MainClass
	{
		[STAThread]
		static void Main(string[] args)
		{
			try
			{
				Subject s = new Subject();
				
				s.f1();				
				s.f2(); // should fail at runtime
			}
			catch (Exception e)
			{
				Console.WriteLine("Exception caught with message: " + e.Message);
				//Console.WriteLine(e.StackTrace);
			}
		}
	}
}
